esmvalcore/cmor/_fixes/cmip6/ukesm1_0_ll.py

```diff
@@ -0,0 +1,33 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
+"""Fixes for UKESM1-0-LL."""
+# import numpy as np
+# import iris
+
+from ..fix import Fix
+
+
+class allvars(Fix):
+    """Fixes for all vars."""
+
+    def fix_metadata(self, cubelist):
+        """
+        Fix non-standard dimension names.
+
+        Parameters
+        ----------
+        cubelist: iris CubeList
+            List of cubes to fix
+
+        Returns
+        -------
+        iris.cube.CubeList
+        """
+        parent_units = 'parent_time_units'
+        bad_value = 'days since 1850-01-01-00-00-00'
+        for cube in cubelist:
+            try:
+                if cube.attributes[parent_units] == bad_value:
+                    cube.attributes[parent_units] = 'days since 1850-01-01'
+            except AttributeError:
```

Comment:
```suggestion
            except KeyError:
```
AttributeError would be raised if `cube.attributes` did not exist, you probably mean KeyError, which would be raised if there is no key `'parent_time_units'` in `cube.attributes`. Maybe it would be simpler to just write
```python
if cube.attributes.get(parent_units, '') == bad_value:
    cube.attributes[parent_units] = 'days since 1850-01-01'
```
without the try except